uptycs/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d27ee4b6-649d-42bd-b7ac-fb40537d7031",
+  "app_id": "uptycs",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect alerts, detections data from Uptycs platform",
+    "title": "Uptycs",
+    "media": [
+     {
+      "media_type": "image",
+      "caption": "Uptycs events dashboard",
+      "image_url": "images/integration_dashboard_1.png"
+     },
+     {
+      "media_type": "image",
+      "caption": "Uptycs events per host trend graph",
+      "image_url": "images/integration_dashboard_2.png"
+     },
+     {
+      "media_type": "image",
+      "caption": "Uptycs detection as a DataDog event",
+      "image_url": "images/data_collected_1.png"
+     },
+     {
+      "media_type": "image",
+      "caption": "Uptycs alert as a DataDog event",
+      "image_url": "images/data_collected_2.png"
+     }
+    ],
+    "classifier_tags": [
+      "Category::Cloud",
+      "Category::Collaboration",
+      "Category::Alerting",
+      "Category::Compliance",
+      "Category::Security",
+      "Offering::Integration",
+      "Submitted Data Type::Events"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "Uptycs Events Dashboard": "assets/dashboards/uptycs.json"
+    },
+    "monitors": {},
```

Comment:
I believe you just need to remove this key entirely to get validations to pass if you don't have any monitors attached to this App. 
```suggestion
```